module/signature/aggregation.go

```diff
@@ -349,7 +360,7 @@ func (p *PublicKeyAggregator) KeyAggregate(signers []int) (crypto.PublicKey, err
 		// remove the missing keys
 		updatedKey, err = crypto.RemoveBLSPublicKeys(updatedKey, missingSignerKeys)
 		if err != nil {
-			// not expected in notrmal operations as there is at least one key, and all keys are BLS
+			// not expected in normal operations as there is at least one key, and all keys are BLS
```

Comment:
```suggestion
			// no error expected as there is at least one key (checked above), and all keys are BLS (checked in the constructor)
```

same suggestion applies to [line 357](https://github.com/onflow/flow-go/blob/35738f2618fb9dd87060955fe5bef0b232341992/module/signature/aggregation.go#L357) (still with the typo "notrmal"): https://github.com/onflow/flow-go/blob/35738f2618fb9dd87060955fe5bef0b232341992/module/signature/aggregation.go#L357